docs/contents.md

```diff
@@ -5,33 +5,39 @@
 
 > **WARNING:** After deployment, this Solution Pack installs or upgrades the stated list of connectors.
 
 ## Global Variable
-| Global Variable | Description |
-| :-              | :-          |
- |`Demo_mode`     | Enables playbook to execute a mock output |
+
+| Global Variable | Description                               |
+|:----------------|:------------------------------------------|
+| `Demo_mode`     | Enables playbook to execute a mock output |
 
 ## Record Sets
-| Scenario          | Description |
-| :-                | :-          |
-|Brute Force Attack | This scenario demonstrates a typical Multiple Login Failure, detected using the **Syslog** connector, on an asset exposed to internet. |
-|Brute Force Attack(FortiSIEM)| This scenario demonstrates a typical Multiple Login Failure, detected using the **FortiSIEM** connector, on an asset exposed to internet. |
+
+| Scenario                      | Description                                                                                                                               |
+|:------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------|
+| Brute Force Attack            | This scenario demonstrates a typical Multiple Login Failure, detected using the **Syslog** connector, on an asset exposed to internet.    |
+| Brute Force Attack(FortiSIEM) | This scenario demonstrates a typical Multiple Login Failure, detected using the **FortiSIEM** connector, on an asset exposed to internet. |
 
 ## Playbook Collections
+
 | 02 - Use Case - Brute Force Attack |
-|:-                                      |
+|:-----------------------------------|
+
+| Playbook Name                                   | Description                                                                                                                                                                |
+|:------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Investigate Brute Force Attempt                 | Investigates login failures and also identifies other impacted assets that have been victims of the brute force attempts from a particular source of attack                |
+| Investigate Brute Force Attempt (FortiSIEM)     | Investigates login failures from FortiSIEM and also identifies other impacted assets that have been victims of the brute force attempts from a particular source of attack |
+| Scenario - FortiSIEM (Brute Force Attack) Alert | Generates a demo alert for Brute Force Attack                                                                                                                              |
```

Comment:
In this we should add the tag "Renamed"

Reply:
Redundant. As we are mentioning the same in release notes.